index.html

```diff
@@ -15,6 +15,14 @@
 <body>
     <section class="head">
         <header class="heading">
+            <nav class="amenu">
+                <span class="xicon"><img src="Images/Xicon.svg" alt="Xicon"></span>
+                <ul class="mmenu">
+                    <li class="mmenu1"><a class="mobile" href="#">Portfolio</a></li>
+                    <li class="mmenu1"><a class="mobile" href="#">About</a></li>
+                    <li class="mmenu1"><a class="mobile" href="#">Contact</a></li>
```

Comment:
- [ ] Your menu or hamburger icon is working properly and that's great 👏. However, when I click on one of the navigation links, it doesn't take me to that particular section while it is among the project requirements. Look at this screenshot showing that point in the requirements 👇 👇 
![image](https://user-images.githubusercontent.com/90222110/179617866-222fc677-d180-4340-a3ea-0202083fc7a7.png)
💡 **HINT:** Please give each section (i.e: portfolio, contact me, about me) a unique ID, then add that id in the corresponding `<a>` tag in the `href` attribute. This will redirect the user to that specific section when they click/tap to the corresponding nav link 😎 👍

Reply:
Thank you for the kind word and the hint.